src/Sentry.Unity.Editor/Batch.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+namespace Sentry.Unity.Editor
+{
+
+    public static class Batch
+    {
+        /// <summary>
+        /// Allows the configuration of Sentry Options using Unity Batch.
+        /// </summary>
+        public static void ConfigureOptions() => ConfigureOptions(ParseCommandLineArguments());
+
+        private static void ConfigureOptions(Dictionary<string, string> args, [CallerMemberName] string functionName = "")
+        {
+            Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, "{0}: Invoking SentryOptions", functionName);
+
+            if (!EditorApplication.ExecuteMenuItem("Tools/Sentry"))
+            {
+                throw new Exception("Menu item Tools -> Sentry was not found.");
+            }
+
+            var optionsWindow = EditorWindow.GetWindow<SentryWindow>();
+            var options = optionsWindow.Options;
+
+            if (options is null)
+            {
+                throw new InvalidOperationException("SentryOptions not found");
+            }
+            Debug.LogFormat(LogType.Log, LogOption.NoStacktrace, null, "{0}: Found SentryOptions", functionName);
```

Comment:
can we use options.Logger?.LogDebug(..)?

Reply:
This is not SentryUnityOptions but ScriptableSentryUnityOptions so Logger isn't available there.